app/views/devise/shared/_links.html.erb

```diff
@@ -6,9 +6,9 @@
   <%= link_to "Sign up", new_registration_path(resource_name) %><br />
 <% end -%>
 
-<%- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations' %>
-  <%= link_to "Forgot your password?", new_password_path(resource_name) %><br />
-<% end -%>
+<!-- # <%- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations' %>
+#   <%= link_to "Forgot your password?", new_password_path(resource_name) %><br />
+# <% end -%> -->
```

Comment:
I remain extremely ambivalent about merging commented-out code such as this into the `master` branch -- _especially_ in view templates, where it will still be rendered to the page as commented-out HTML.